packages/app/pages/checkout/[network]/[address].tsx

```diff
@@ -258,7 +317,7 @@ export const getStaticPaths: GetStaticPaths<QueryParams> = () => {
   });
 };
 
-export const getServerSideProps: GetServerSideProps<PageProps, QueryParams> = (
+export const getStaticProps: GetStaticProps<PageProps, QueryParams> = (
```

Comment:
I had to revert this because I was seeing this error:

![Screenshot 2022-02-03 at 11 19 33 PM](https://user-images.githubusercontent.com/97116159/152399690-f4a59767-24a1-4900-8f4c-c3bfbdc2f2a6.png)

Reply:
agh my bad. I'll fix this on my end, I think we might be able to just nuke `getStaticPaths`